### ISSUE TITLE
Deliver database structure and data to the BIM IT team.

### ISSUE THREAD

**Issue by damianooldoni** [inbo/speciesbim#32]
We work on a copy of the database which is saved on an Amazon S3 of INBO. 
How to transfer structure and data to BIM IT team?

**Comment by niconoe** [comment]
IMHO we don't transfer them the database, we gave them access to the scripts that transform the database into the new version. Those scripts already allow to change the database structure (to be called once) and to populate/repopulate the new tables automatically (to be called as often as they want).

**Comment by damianooldoni** [comment]
Yes, indeed! Thanks @niconoe to point this out! I just copy-pasted the task as referred in the kick-off meeting.
The only thing which should be adapted by them is the section with server infos in the config.ini. 

Probably this is the easiest issue ever, but I would like to mention it as it is one of the tasks to be accomplished and it has to be done in collaboration with BIM IT team of course.

**Comment by niconoe** [comment]
We could ask them to soon provide a new copy of their database, just to make sure there are no recent changes that would break compatibility with our scripts.

**Comment by damianooldoni** [comment]
That's a good idea @niconoe : I add it to the Google Doc as reminder.

**Comment by damianooldoni** [comment]
BIM suggest to test our work on a development version of their db on Paola(?). This is a very good news for us as well because we can test our implementation on a constantly updated version of the db. @billobah: did I formulate it correctly?

BIM confirms also the choice of Python (scripts/notebooks).

**Comment by billobah** [comment]
Like I said it during our meeting, we plan to build 3 Servers: 1 for development, 1 for staging and 1 for production tasks. May be the test could be conducted in the development server. We will discuss about this with Mathias and he will come back to you about it.

Regards

**Comment by damianooldoni** [comment]
Thanks @billobah. It would be nice to add it to draft report I sent to BIM yesterday.

**Comment by damianooldoni** [comment]
@billobah: do you have an idea about the timeline for the plans you mentioned in https://github.com/inbo/speciesbim/issues/32#issuecomment-688914225?

**Comment by billobah** [comment]
Hello the INBO team.
Last Thursday, we had a meeting with Mathias in order to clarify the role of the IT team in the framework of the DB Biodiversity Project. Here after the summary of the actions to be taken regarding our involvement.

**Our Git platform**
We suggest you to use our Git platform (https://git.environnement.brussels) to exchange with us. We will create an account for you three (with your e-mail as login) in order to get access to our git plateform. Once access is granted, could you put the project’s codes/scripts in our git?  You will have access to the “speciesbim” project created in our platform. 

**Using the scripts/codes you developped**
Could we have a technical meeting in order to discuss some technical points as:
- using the scripts (the jupiter Notebook demo, etc.)
- availability of technical documentation explaining the scripts, etc.
- DB updating process (SQL Queries, etc.)
- impact of the updating process on the DB structure (defined functions, tables, etc.)
- etc.

Please see with Mathias for the planning of this meeting.

I will make you informed once your accounts are created in our git.

Best regards

Billo

**Comment by niconoe** [comment]
Thanks for your input, @billobah!

Please note that for the moment we can't reach at all the URL of your Git platform, is it accessible from the outside (of IBGE / BIM)?

![Screenshot 2020-09-15 at 15 55 53](https://user-images.githubusercontent.com/386387/93219953-3707ad00-f76c-11ea-8e36-cf087d0a090b.png)

**Comment by billobah** [comment]
Hello @niconoe, @damianooldoni, @peterdesmet 

We have to give you first the access to our git platform (may be through VPN, I don't know exactly). The person in charge of that issue is at holidays. I will comeback to you as soon as your access is created, may be before the end of september.

**Comment by niconoe** [comment]
Hey @billobah, thanks for the reply. Your git platform is based on GitLab, right?

In that case, we might make use of the [import project feature](https://docs.gitlab.com/ee/user/project/import/github.html) so we not only import the git repository itself, but also the GitHub issues (and possibly PR and others) so we have the full project history for the long-term maintenance.

**Comment by billobah** [comment]
Hello @niconoe,

Yes, we have a GitLab platform. Good idea for important the repo and the issues. In that way, we can directly interact through our platform, and my colleagues (Fayrouze and Alfonso) could help to answer to the issues.

**Comment by damianooldoni** [comment]
@niconoe and I succeeded to access your GitLab environment (https://git.environnement.brussels/) using the FortiClient VPN program and the credentials we got on Friday, Oct 2. As mentioned by @niconoe in https://github.com/inbo/speciesbim/issues/32#issuecomment-693431695 we should make use of the import project feature to move everything we can move from this repo. I will add this to agenda of Wednesday.

**Comment by peterdesmet** [comment]
I have access too now. 👍

**Comment by niconoe** [comment]
This repository (code, issues, pull request, ...) has now been moved to https://git.environnement.brussels/.

Please continue development there.
